mobile/library/common/extensions/cert_validator/platform_bridge/platform_bridge_cert_validator.cc

```diff
@@ -84,10 +88,19 @@ ValidationResults PlatformBridgeCertValidator::doVerifyCertChain(
 
   ValidationJob job;
   job.result_callback_ = std::move(callback);
-  job.validation_thread_ =
-      std::thread(&verifyCertChainByPlatform, &(job.result_callback_->dispatcher()),
-                  std::move(certs), std::string(host), std::move(subject_alt_names), this);
-  std::thread::id thread_id = job.validation_thread_.get_id();
+  Event::Dispatcher& dispatcher = job.result_callback_->dispatcher();
+  job.validation_thread_ = thread_factory_->createThread(
+      [&, certs = std::move(certs), host = std::string(host),
```

Comment:
do we need to capture everything by reference if we are already passing in certs, hosts, etc?  looks like we only need to capture `this` and `dispatcher` in addition, so we could explicitly capture them instead of `&` (since we're already explicitly creating capture variables for everything else)

Reply:
Makes sense. Done.